package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Framework for building front-end apps",
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
```

Comment:
The `--hoist` flag is needed due to https://github.com/Microsoft/TypeScript/issues/20945